webpack.common.js

```diff
@@ -11,7 +11,8 @@ module.exports = {
 		'admin/stun-server': path.join(__dirname, 'src', 'StunServerSettings.js'),
 		'admin/turn-server': path.join(__dirname, 'src', 'TurnServerSettings.js'),
 		'collections': path.join(__dirname, 'src', 'collections.js'),
-		'talk': path.join(__dirname, 'src', 'main.js')
+		'talk': path.join(__dirname, 'src', 'main.js'),
+		'appSidebarTab': path.join(__dirname, 'src', 'TalkSidebarTab.js'),
```

Comment:
kebab case for files as it's usually more readable :woman_shrugging: 
```suggestion
		'app-sidebar-tab': path.join(__dirname, 'src', 'TalkSidebarTab.js'),
```

Reply:
Vue files are camelCased, do you refer to .js files only here?

Reply:
yes :)

Reply:
just the compiled name